src/components/footer/footer.yaml

```diff
@@ -125,6 +147,16 @@ examples:
     meta:
       text: GOV.UK Prototype Kit v7.0.1
 
+- name: with custom licence text
+  description: Custom licence section
+  data:
+    licence: GOVUK Licence
```

Comment:
Hi @georgeeker, this will need to have a `.text` or `.html` property to work.

See the preview: https://govuk-frontend-review-pr-1314.herokuapp.com/components/footer/with-custom-licence-text/preview

It's not triggering `{% if params.licence.html or params.licence.text %}`